nautobot/extras/tests/test_views.py

```diff
@@ -1,13 +1,20 @@
+from datetime import datetime, timedelta
+import os.path
 import urllib.parse
 import uuid
 
+from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.contenttypes.models import ContentType
+from django.test import override_settings, SimpleTestCase
```

Comment:
Based on the Travis failure, I think you need a `from django.http import Http404` somewhere in here.

Reply:
Ah, good catch, thanks!